indent/ps1.vim

```diff
@@ -1,20 +1,96 @@
 " Vim indent file
-" Language:           Windows PowerShell
-" Maintainer:         Peter Provost <peter@provost.org>
-" Version:            2.10
-" Project Repository: https://github.com/PProvost/vim-ps1
-" Vim Script Page:    http://www.vim.org/scripts/script.php?script_id=1327"
+" Language:		PowerShell
+" Maintainer:	Lior Elia - http://blogs.msdn.com/lior
```

Comment:
If requesting to merge, please maintain the header information. You can add your name to the list of maintainers as other scripts have done.